tsmicroctl.c

```diff
@@ -82,8 +84,14 @@ int main(int argc, char *argv[])
 	int opt_current = -1;
 	int opt_sleep = -1;
 
+	board = get_board();
+	if (!board) {
+		printf("Unsupported board\n");
```

Comment:
Can we change this to "platform?" The user of board should be minimized since some of our products are no longer single boards and "board" is more informal. The variable and function `get_board()` are fine since they are pretty internal, but, we should be printing matching our documentation terminology.